cryptus-app/components/wallet_viewer/profile_wallet_viewer/collection_block/collection_nfts/collection_nfts.module.scss

```diff
@@ -17,32 +15,23 @@
   border-width: 2px;
   border-radius: 10px;
   border-color: #333333;
-  // padding: min(10%, 128px);
   white-space: nowrap;
   width: 150px;
   height: 150px;
 }
 
 .grid2x2 {
-  height: 100%;
-  width: 100%;
-  display: flex;
-  flex-wrap: wrap;
-  flex-direction: row;
-  justify-content: space-evenly;
+  width:100%;
+  height:100%;
+  display:grid;
+  grid-template-columns: 50% 50%;
+  grid-column-gap: 0;
+  grid-row-gap: 0;
 }
 
 .grid2x2 > div {
-  display: flex; 
-  flex-basis: calc(50%);  
-  justify-content: center;
-  flex-direction: column;
-}
-
-.grid2x2 > div > div {
-  display: flex;
-  justify-content: center;
-  flex-direction: row;
+  align-self: center;
+  justify-self: center;
```

Comment:
don't use align-self or justify-self, the logic of where a children is placed should be in the parent

Reply:
I need to place what is in the container and not the container. That's why I need to specify to the parent class that only it's first children layer need to have them